scheme/tpm-enacttrust/test/corim/build-test-vectors.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# Copyright 2022-2024 Contributors to the Veraison project.
+# SPDX-License-Identifier: Apache-2.0
+
+set -eu
+set -o pipefail
+
+THIS_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+CORIM_TEMPLATE=corimMini
+
+COMID_TEMPLATES=(
+	ComidTpmEnactTrustAKOne
+	ComidTpmEnactTrustGoldenOne
+	ComidTpmEnactTrustAKMult
+	ComidTpmEnactTrustBadInst
+	ComidTpmEnactTrustNoInst
+	ComidTpmEnactTrustMultDigest
+	ComidTpmEnactTrustGoldenTwo
+	ComidTpmEnactTrustNoDigest
+	ComidTpmEnactTrustAKBadInst
+)
+
+# function generate_go_test_vector constructs CBOR test vector using
+# supplied comid and corim json template and saves them in a file
+# $1 file name for comid json template, example one of COMID_TEMPLATES
+# $2 file name for corim json template, example CORIM_TEMPLATE
+# $3 a qualifier for each cbor test vector name
+generate_go_test_vector () {
+	local comid=$1
+	local corim=$2
+	local qualifier=$3
+
+	echo "generating test vector using $comid $corim"
+	cocli comid create -t "$THIS_DIR/src/$comid.json" -o "$THIS_DIR"
+	cocli corim create -m "$THIS_DIR/$comid.cbor" -t "$THIS_DIR/src/$corim.json" \
+		-o "$THIS_DIR/$qualifier${corim^}${comid^}.cbor"
+	rm "$THIS_DIR/$comid.cbor"
+}
```

Comment:
There are 5 identical copies of this function.  Probably worth factoring it out under `scripts/`

Reply:
This function is probably too small and context-specific (it relies on being run with in a particular directory structure) to make it worth it for it alone. Maybe if we wanted to have more of a unified harness for scheme tests, this would become part of that.  But that's a larger refactor, and better off done as a separate change.

Reply:
That bigger refactoring could start with a small step - adding the function under a `scheme/scripts` directory ;-)

Reply:
Done.

Reply:
Awesome, thanks for putting up with a grumbling geezer!